api/events/eventsRouter.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const todo = require('./eventsModel');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  todo
+    .get()
```

Comment:
Just wondering if this endpoint has been tested. I did not see where 'stuff' was defined and it is not present in the argument list of your main function